jetsocat/src/main.rs

```diff
@@ -113,13 +114,33 @@ fn listen_command() -> Command {
         .usage(format!("{} listen BINDING_ADDRESS", env!("CARGO_PKG_NAME")))
         .action(listen_action);
 
-    apply_common_flags(apply_server_side_flags(cmd))
+    apply_common_flags(apply_pipe_server_flags(cmd))
 }
 
 pub fn listen_action(c: &Context) {
     let res = ServerArgs::parse("listen", c).and_then(|args| {
         let log = setup_logger(args.common.logging);
-        run(log.clone(), jetsocat::server::listen(args.common.addr, args.pipe, log))
+        run(log.clone(), jetsocat::server::listen(args.common.addr, args.cmd, log))
+    });
+    exit(res);
+}
+
+fn proxy_command() -> Command {
+    let cmd = Command::new("tcp-proxy")
```

Comment:
Since this is mostly specific code for RDP rendezvous mode, I suggest to rename "tcp-proxy" to "rdp-tunnel" or "rdp-forward" for now. At least I'm in favor of using "tunnel" or "forward" instead of "proxy" (possible confusion with SOCKS/HTTP proxy also supported by jetsocat)

Reply:
The current code in jetsocat for "tcp-proxy" could handle any tcp port handling, so maybe it is better to rename it to "tcp-tunnel-accept"? What do you think?

Reply:
I asked Marc-André about whether we should make it rdp specific or not, I think he will clarify this soon. If we keep it general over TCP then `tcp-tunnel-accept` sounds good to me! :+1:

Reply:
@pacmancoder Hi! I discussed with Marc-André, we agreed on "jet-tcp-accept" :slightly_smiling_face: